cBinder/WrapperBuilder.py

```diff
@@ -245,22 +250,19 @@ def _build_python_wrapper_for_function(self, writer, module_name, function):
 
             parameter_list = []
             for param in function.parameters:
+                str1 = param.name + unique_identifier_suffix
```

Comment:
if this is to be extracted into a variable then there are 4 more places below where this is used. Could use a better name too, `fully_qualified_name` maybe?